apps/hostel/models.py

```diff
@@ -3,6 +3,7 @@
 from sqlalchemy import (MetaData, Column, INTEGER, VARCHAR, ForeignKey, FLOAT, Table)
 from sqlalchemy.orm import relationship
 
+
 metadata_obj = MetaData()
```

Comment:
Don't need to user metadata anywhere outside main metadata (db.py)

Reply:
Can't understand your question. Does it concerns to my ticket?